docs/rules/spaced-comment.md

```diff
@@ -21,8 +21,8 @@ The rule takes two options.
 
 * This rule can also take a 2nd option, an object with any of the following keys: `"exceptions"` and `"markers"`.
 
-    * The `"exceptions"` value is an array of string patterns which are considered exceptions to the rule.
-    Please note that exceptions are ignored if the first argument is `"never"`.
+    * The `"exceptions"` value is an array of string patterns which are considered exceptions to the rule. The rule will not warn when the pattern starts from the beginning of the first line and repeats until the end of the line or `*/` if the comment is a single line comment.
```

Comment:
Comment doesn't have to be at the beginning of a line, so it might be more correct to say "when the pattern starts from the beginning of the comment".